packages/binding-modbus/src/modbus-client.ts

```diff
@@ -243,31 +247,31 @@ export default class ModbusClient implements ProtocolClient {
         const mode = contentLength > 0 ? "w" : "r";
 
         // Use form values if provided, otherwise use form values (we are more merciful then the spec for retro-compatibility)
```

Comment:
```suggestion
        // Use URI values if provided, otherwise use form values (we are more merciful then the spec for retro-compatibility)
```
? Or even get rid of this comment since we are not that merciful anymore 😆

Reply:
I have adjusted it completely